spec/services/rate_limit_checker_spec.rb

```diff
@@ -14,8 +14,14 @@ def cache_key(action)
       expect(rate_limit_checker.limit_by_action("random-nothing")).to be(false)
     end
 
+    it "raises an error if user does not have an ID" do
+      action = described_class::ACTION_LIMITERS.keys.first
+      limiter = described_class.new(build(:user))
```

Comment:
lol this is one of those few times when `build` is truly the best tool for the job! 💪